packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -36,7 +36,7 @@ module.exports = (resolve, rootDir) => {
     testURL: 'http://localhost',
     transform: {
       '^.+\\.css$': resolve('config/jest/cssTransform.js'),
-      '^.+\\.tsx?$': resolve('config/jest/typescriptTransform.js'),
+      '^.+\\.tsx?$': resolve('node_modules/ts-jest/preprocessor.js'),
```

Comment:
This will be `node_modules/react-scripts-ts/ts-jest/preprocessor.js` - at least until the user has ejected. Which I think we'll need to consider?